doc/build/dts/intro-syntax-structure.rst

```diff
@@ -0,0 +1,472 @@
+    in the previous section has I2C address 0x39.
+
+SPI peripherals
+    An index representing the peripheral's chip select line number.
+    (If there is no chip select line, 0 is used.)
+
+Memory
+    The physical start address.
+    For example, a node named ``memory@2000000`` represents RAM starting at
+    physical address 0x2000000.
+
+Memory-mapped flash
+    Like RAM, the physical start address.
+    For example, a node named ``flash@8000000`` represents a flash device
+    whose physical start address is 0x8000000.
+
+Fixed flash partitions
+    This applies when the devicetree is used to store a flash partition table.
+    The unit address is the partition's start offset within the flash memory.
+    For example, take this flash device and its partitions:
+
+    .. code-block:: devicetree
+
+        flash@8000000 {
+            /* ... */
+            partitions {
+                    partition@0 { /* ... */ };
+                    partition@20000 {  /* ... */ };
+                    /* ... */
+            };
+        };
+
+    The node named ``partition@0`` has offset 0 from the start of its flash
+    device, so its base address is 0x8000000. Similarly, the base address of
+    the node named ``partition@20000`` is 0x8020000.
+
+.. _dt-important-props:
+
+Important properties
+********************
+
+.. Documentation maintainers: If you add a property to this list,
+   make sure it gets linked to from gen_devicetree_rest.py too.
+
+The devicetree specification defines several standard properties.
+Some of the most important ones are:
+
+compatible
+    The name of the hardware device the node represents.
+
+    The recommended format is ``"vendor,device"``, like ``"avago,apds9960"``,
+    or a sequence of these, like ``"ti,hdc", "ti,hdc1010"``. The ``vendor``
+    part is an abbreviated name of the vendor. The file
+    :zephyr_file:`dts/bindings/vendor-prefixes.txt` contains a list of commonly
+    accepted ``vendor`` names. The ``device`` part is usually taken from the
+    datasheet.
+
+    It is also sometimes a value like ``gpio-keys``, ``mmio-sram``, or
+    ``fixed-clock`` when the hardware's behavior is generic.
+
+    The build system uses the compatible property to find the right
+    :ref:`bindings <dt-bindings>` for the node. Device drivers use
+    ``devicetree.h`` to find nodes with relevant compatibles, in order to
+    determine the available hardware to manage.
+
+    The ``compatible`` property can have multiple values. Additional values are
+    useful when the device is a specific instance of a more general family, to
+    allow the system to match from most- to least-specific device drivers.
+
+    Within Zephyr's bindings syntax, this property has type ``string-array``.
+
+reg
+    Information used to address the device. The value is specific to the device
+    (i.e. is different depending on the compatible property).
+
+    The ``reg`` property is a sequence of ``(address, length)`` pairs. Each
```

Comment:
Worth mentioning that it is in hex, e.g. for i2c if someone is looking at a datasheet, they might not be aware that they need to put in 1B and not 27